src/main/java/seedu/address/logic/commands/FacebookPostCommand.java

```diff
@@ -30,22 +35,34 @@ public FacebookPostCommand(String message) {
         toPost = message;
     }
 
-    @Override
-    public CommandResult execute() throws CommandException {
-        if (!FacebookConnectCommand.isAuthenticated()) {
-            FacebookConnectCommand newFacebookConnect = new FacebookConnectCommand();
-            newFacebookConnect.execute();
-        }
-
+    /**
+     * Completes the post command
+     * @throws CommandException
+     */
+    public static void completePost() throws CommandException {
         Facebook facebookInstance = FacebookConnectCommand.getFacebookInstance();
-        String user = "";
+        user = "";
```

Comment:
Think it might be better to make it null than an empty string.

Reply:
ok noted, thanks!